### PR TITLE
initial

### PR DESCRIPTION
**Pull request by axtens**

### CODE REVIEW

**Comment by KTSnowy** [comment]
Hey @axtens, the test.sh needs to be updated to match the other exercises.

**Comment by KTSnowy** [comment]
Capture
The Linux CI failed because of the missing `bash` added in the updated version of the script. [Here, copy this one](https://github.com/exercism/cobol/blob/main/exercises/practice/armstrong-numbers/test.sh)

File: exercises/practice/triangle/src/triangle.cob [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+       ENVIRONMENT DIVISION.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+      *Incoming
+       01 WS-SIDES PIC X(20).
+       01 WS-PROPERTY PIC X(11).
+      *Outgoing
+       01 WS-RESULT PIC 9.
+       PROCEDURE DIVISION.
+       TRIANGLE.
```
Comment (kotp) [inline exercises/practice/triangle/src/triangle.cob:12]: Missing an EOL for what should be line 12.

File: exercises/practice/triangle/bin/fetch-cobolcheck.ps1 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    $latestUrl = "https://api.github.com/repos/0xE282B0/cobol-check/releases/latest"
+    Invoke-RestMethod -Uri $latestUrl -PreserveAuthorizationOnRedirect @requestOpts
+    | Select-Object -ExpandProperty assets
+    | Where-Object { $_.browser_download_url -match $FileName }
+    | Select-Object -ExpandProperty browser_download_url
+}
+
+$downloadUrl = Get-DownloadUrl
+$outputFile = Join-Path -Path $PSScriptRoot -ChildPath "cobolcheck.exe"
+Invoke-WebRequest -Uri $downloadUrl -OutFile $outputFile @requestOpts
```
Comment (kotp) [inline exercises/practice/triangle/bin/fetch-cobolcheck.ps1:28]: Missing EOL for what should be line 28.

**Review by kotp — CHANGES_REQUESTED** [review]

File: exercises/practice/triangle/bin/fetch-cobolcheck.ps1 (hunk 2 above)
Comment (axtens) [reply exercises/practice/triangle/bin/fetch-cobolcheck.ps1]: Thank you

**Review by KTSnowy — APPROVED** [review]

**Pull request merged**